optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -213,6 +216,63 @@ private void applySolverProperties(RecorderContext recorderContext,
                 .collect(Collectors.toList());
     }
 
+    private void stopBuildIfClassesExistThatHavingPlanningAnnotationsButAreNotAPlanningEntity(SolverConfig solverConfig,
```

Comment:
Naming: stopBuildIf...Not... => assert...
See AbstractScoreDirector assert methods.

Reply:
Let's see if we can make it shorter. How about assertNoMemberAnnotationWithoutClassAnnotation() ? Or assertNoOrphanMemberAnnotation? Although "orphan" is not a common term for this.

Naming is difficult and very important for long-term projects like OptaPlanner.